packages/cmrjs/src/cmr-utils.js

```diff
@@ -113,15 +113,19 @@ function granulesToCmrFileObjects(granules) {
  * @param {string} cmrFile.filename - the s3 uri to the cmr xml file
  * @param {string} cmrFile.metadata - granule xml document
  * @param {Object} cmrClient - a CMR instance
+ * @param {string} revisionId - Optional CMR Revision ID
  * @returns {Object} CMR's success response which includes the concept-id
  */
-async function publishECHO10XML2CMR(cmrFile, cmrClient) {
+async function publishECHO10XML2CMR(cmrFile, cmrClient, revisionId) {
   const builder = new xml2js.Builder();
   const xml = builder.buildObject(cmrFile.metadataObject);
-  const res = await cmrClient.ingestGranule(xml);
+  const res = await cmrClient.ingestGranule(xml, revisionId);
   const conceptId = res.result['concept-id'];
+  const cmrRevisionId = res.result['revision-id'];
```

Comment:
Does a `revision-id` come back in the result from CMR? If so, what is the reason for using that one over the one provided as an argument? 

If the specified revisionId was wrong, then CMR would error on line 122 and not make it to the log, as I understand it. So this seems slightly unnecessary unless I'm missing something.

Reply:
Oh, that's a good point. I suppose if they are technically the same, we could just use the revision ID the user provides.